training_scripts/gsgnn_np/ml_nc_utext_glem.yml

```diff
@@ -24,13 +24,20 @@ gsf:
     fanout: "4"
     num_layers: 1
     hidden_size: 128
-    use_mini_batch_infer: true    
+    use_mini_batch_infer: false
     training_method:
       name: glem
       kwargs:
         em_order_gnn_first: false
         inference_using_gnn: true
         pl_weight: 0.5
+        num_pretrain_epochs: 1
+        lm_param_group:
+          - pure_lm
+        gnn_param_group:
+          - node_input_projs
+          - node_proj_matrix
+          - sparse_embed
```

Comment:
i wonder if we should have users to configure this. shouldn't we optimize the parameters like this? are there any other configurations to optimize the model parameters?

Reply:
The best settings of these parameters depends on the dataset and task. These are all the hparams for GLEM: 
```
        em_order_gnn_first
        inference_using_gnn
        pl_weight
        num_pretrain_epochs
        lm_param_group
        gnn_param_group
```

Reply:
you need to update this as well